givitsite/templates/feed.html

```diff
@@ -0,0 +1,30 @@
+{%extends 'base.html'%}
+
+{%block content%}
+
+<h1>Feed</h1>
+<table border='1'>
+<tr>
+<td>Title</td>
+<td>City</td>
+<td>URL</td>
+<td>Phone Number</td>
+<td>Pic</td>
+<td>Match</td>
+</tr>
+{% for item in founds%}
+
+<tr>
+<td>{{item.title}}</td>
+<td>{{item.city}}</td>
+<td><a href="{{item.url}}">link</a></td>
+<td>{{item.phone_number}}</td>
+<td><img src="{{item.picture}}" width='50px'/></td>
+<td><input type='button' value='match'/></td>
```

Comment:
I am not sure what does this button do :thinking:

Reply:
in the future, clicking on this button will change the status of the item.
it's a preparation for the new feature

Reply:
Can you change `<input type='button'...` to `<button type='button' class='btn btn-primary'>Change status</button>`?

Reply:
I am not a fan of adding this "preparation" but if you find it necessary then maybe you can add a comment above it with the Github URL to the ticket that will handle this.

Reply:
fixed